docs/getting-started/dev-setup.md

```diff
@@ -0,0 +1,207 @@
+        - MYSQL_DATABASE=test_sh
+      command: --wait_timeout=2592000 --interactive_timeout=2592000 --max_connections=300
+        
+    elasticsearch:
+      image: bitergia/elasticsearch:6.8.6-secured
+      command: elasticsearch -Enetwork.bind_host=0.0.0.0 -Ehttp.max_content_length=2000mb
+      ports:
+        - 9200:9200
+      environment:
+        - ES_JAVA_OPTS=-Xms2g -Xmx2g
+    kibiter:
+      restart: on-failure:5
+      image: bitergia/kibiter:secured-v6.8.6-3
+      environment:
+        - PROJECT_NAME=Demo
+        - NODE_OPTIONS=--max-old-space-size=1000
+        - ELASTICSEARCH_USER=kibanaserver
+        - ELASTICSEARCH_PASSWORD=kibanaserver
+        - ELASTICSEARCH_URL=["https://elasticsearch:9200"]
+        - LOGIN_SUBTITLE=If you have forgotten your username or password ...
+      links:
+        - elasticsearch
+      ports:
+        - 5601:5601
+```
+
+#### docker-compose (without SearchGuard) [&uarr;](#getting-the-containers-)
+
+Note: Here, access to kibiter and elasticsearch don't need credentials.
+
+```
+version: '2.2'
+services:
+    elasticsearch:
+      image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.8.6
+      command: elasticsearch -Enetwork.bind_host=0.0.0.0 -Ehttp.max_content_length=2000mb
+      ports:
+        - 9200:9200
+      environment:
+        - ES_JAVA_OPTS=-Xms2g -Xmx2g
+        - ANONYMOUS_USER=true
+    kibiter:
+      restart: on-failure:5
+      image: bitergia/kibiter:community-v6.8.6-3
+      environment:
+        - PROJECT_NAME=Demo
+        - NODE_OPTIONS=--max-old-space-size=1000
+        - ELASTICSEARCH_URL=http://elasticsearch:9200
+      links:
+        - elasticsearch
+      ports:
+        - 5601:5601
+    mariadb:
+      restart: on-failure:5
+      image: mariadb:10.0
+      expose:
+        - "3306"
+      ports:
+        - "3306:3306"
+      environment:
+        - MYSQL_ROOT_PASSWORD=
+        - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+        - MYSQL_DATABASE=test_sh
+      command: --wait_timeout=2592000 --interactive_timeout=2592000 --max_connections=300
+```
+
+Save the above into a docker-compose.yml file and run
+```
+$ docker-compose up -d
+```
+to get ElasticSearch, Kibiter and MariaDB running on your system.
+
+### Cloning the repositories [&uarr;](#source-code-and-docker-)
+
+In the next step, you will need to fork all the GitHub repos below and clone them to a target local folder (e.g., `sources`).
+
+- [SirModred](https://github.com/chaoss/grimoirelab-sirmordred)
+- [ELK](https://github.com/chaoss/grimoirelab-elk)
+- [Graal](https://github.com/chaoss/grimoirelab-graal)
+- [Perceval](https://github.com/chaoss/grimoirelab-perceval)
+- [Perceval for Mozilla](https://github.com/chaoss/grimoirelab-perceval-mozilla)
+- [Perceval for OPNFV](https://github.com/chaoss/grimoirelab-perceval-opnfv)
+- [Perceval for Puppet](https://github.com/chaoss/grimoirelab-perceval-puppet)
+- [Perceval for FINOS](https://github.com/Bitergia/grimoirelab-perceval-finos)
```

Comment:
```suggestion
- [Perceval for Weblate](https://github.com/chaoss/grimoirelab-perceval-weblate)
```

FINOS is deprecated and will be removed soon. Weblate is added to GrimoireLab. So, it would be great if we can update it.